File: 1651-shuffle-string/1651-shuffle-string.js
/**
 * @param {string} s
 * @param {number[]} indices
 * @return {string}
 */
var restoreString = function(s, indices) {
    var str=[];
    for(i=0;i<=s.length;i++){
        str[indices[i]]=s[i]
    }
    return(str.join(""))
};

File: 1651-shuffle-string/1651-shuffle-string.cs
public class Solution {
    public string RestoreString(string s, int[] indices) {
        char[] str=new char[s.Length];
        for(int i=0;i<s.Length;i++){
            str[indices[i]]=s[i];
        }
        return new string(str);
    }
}